### DISCUSSION TITLE
NO follower data

### DISCUSSION THREAD

**Discussion by Kustomwireless** [NightscoutFoundation/xDrip#3768] [General]
Anyone else not getting Data on there Followers this afternoon? Both of my wife's followers are not getting data this afternoon latest nightly with Xdrip cloud on!

**Comment by Navid200** [comment]
Everything we know about xDrip sync is here:
https://github.com/NightscoutFoundation/xDrip/discussions/3664  

You can try disabling `Use xDrip cloud` on both the master and all followers.  
It may work then. 

Thanks for reporting.
We will need to wait for more updates.

**Reply by AliQ101** [reply]
The same here: XDrip Cloud ON with Xdrip Cloud: Not connected, Status: Waiting for reconnect for more than 3 hours for both Master/follower.

**Reply by Navid200** [reply]
Does it work if you disable `Use xDrip Cloud`?

**Reply by AliQ101** [reply]
Yes, it works if disabled on master/follower.

**Reply by AliQ101** [reply]
Start to work also cloud services with Cloud ON now.

**Reply by dbames** [reply]
We experienced this for a few hours (I think) between a master and two followers.  The followers generally showed they weren't connected to xDrip Cloud.  I tinkered a bit on one of the two followers, but without any success.  "Magically", it resolved itself a while later on and everything spring back into life.

**Comment by fedemotta** [comment]
Same issue here with multiple xDrip users (most of them from Argentina). No follower data using "use xDrip cloud". Only working as a Nightscout follower

**Reply by fedemotta** [reply]
Error reporting started 3 or 4 hs ago
